docker-compose.yml

```diff
@@ -39,3 +39,12 @@ services:
       - .:/app
     environment:
       - JOB_WORKER_URL=redis://redis:6379/0
+  mailman:
+    build: .
+    command: cd script && mailman
```

Comment:
not `mailman_server`?

Reply:
https://github.com/mailman/mailman/blob/master/USER_GUIDE.md

Reply:
Yes you are right! 😅. It should be mailman_server.

Reply:
Let me confirm first. Sorry!